templates/blog/newsletter-form.html

```diff
@@ -0,0 +1,86 @@
+            <label for="insightscloudserver">Cloud and Server</label>
+          </li>
+  
+          <li class="p-list__item u-clearfix">
+            <input type="checkbox" id="insightsdesktop" name="insightsdesktop" value="true" />
+            <label for="insightsdesktop">Desktop</label>
+          </li>
+  
+          <li class="p-list__item u-clearfix">
+            <input type="checkbox" id="insightsiot" name="insightsiot" value="true" />
+            <label for="insightsiot">Internet of Things</label>
+          </li>
+  
+          <li class="p-list__item u-clearfix">
+            <input id="insightsrobotics" name="insightsrobotics" value="true" type="checkbox">
+            <label for="insightsrobotics">Robotics</label>
+          </li>
+  
+          <li class="p-list__item u-clearfix">
+            <input id="insightstutorials" name="insightstutorials" value="true" type="checkbox">
+            <label for="insightstutorials">Tutorials</label>
+          </li>
+          {# These are honey pot fields to catch bots #}
+          <li class="u-off-screen">
+            <label class="website" for="website">Website:</label>
+            <input name="website" type="text" class="website" autocomplete="off" value="" id="website" tabindex="-1" />
+          </li>
+          <li class="u-off-screen">
+            <label class="name" for="name">Name:</label>
+            <input name="name" type="text" class="name" autocomplete="off" value="" id="name" tabindex="-1" />
+          </li>
+          {# End of honey pots #}
+        </ul>
+  
+        <div>
+          <label>
+            <span class="u-no-margin--top">Work email: <span>*</span></span>
+            <input required id="email" name="email" maxlength="255" type="email" pattern="^[^ ]+@[^ ]+\.[a-z]{2,26}$"/>
+          </label>
+        </div>
+  
+        <div>
+          <span>
+            <input class="u-no-margin--top" name="canonicalUpdatesOptIn" id="CanonicalUpdatesOptIn" value="yes" type="checkbox" />
+            <label for="CanonicalUpdatesOptIn" class="p-form__label u-no-margin--top">
+              <small>I agree to receive information about Canonical's products and services.</small>
+            </label>
+          </span>
+        </div>
+        <p>
+          <small>In submitting this form, I confirm that I have read and agree to <a href="/legal/data-privacy/newsletter">Canonical's Privacy Notice</a> and <a href="/legal/data-privacy">Privacy Policy</a>.</small>
+        </p>
+  
+        <div>
+          <span class="p-card--content">
+            <button type="submit" class="u-no-margin--bottom">Subscribe now</button>
+          </span>
+  
+          <input value="1212" name="formid" type="hidden">
+          <input type="hidden" name="Consent_to_Processing__c" value="yes" />
+          <input type="hidden" aria-hidden="true" aria-label="hidden field" name="returnURL" value="/blog#success" />
```

Comment:
This return URL has no message in the header

Reply:
Fixed